examples/video/outstream/pb-ve-outstream-basic-ima.html

```diff
@@ -0,0 +1,100 @@
+---
+layout: video_sample
+title: Prebid Video | Video Outstream Basic Example with Google IMA SDK
+description: An example of an outstream video ad using Prebid.js and the Google IMA SDK.
+videoType: pb-os-basic-ima
+isVideo: true
+sidebarType: 4
+code_height: 2300
+---
+
+<script async src="//cdn.jsdelivr.net/npm/prebid.js@latest/dist/not-for-prod/prebid.js"></script>
+<div class="container">
+    <div class="row">
+        <div class="vidHeader" style="width:75vw;">
+            <h1>{{ page.title }}</h1>
+            <p>{{ page.description }}</p>
+        </div>
+
+        <!--video warning-->
+        <div class="pb-alert pb-alert-important">
+            <p style="color:#85720f"><b>Important:</b>
+            This example uses a test version of Prebid.js hosted on our CDN that is not recommended for production use. It includes all available adapters. Production implementations should build from source or customize the build using the Download page to make sure only the necessary bidder adapters are included.</p>
+        </div>
+
+        <!--header code example-->
+        <div class="pb-alert pb-alert-warning">
+            <p style="color:#a94443"><b>Warning:</b>
+            Do not forget to exchange the placementId in the code examples with your own placementId!</p>
+        </div>
+
+        <div id='adContainer'></div>
+
+        <div class="row">
+           <div class="pb-code-hl-wrap">
+             <pre id='codeExample' class="pb-code-hl"></pre>
+           </div>
+        </div>
+    </div>
+</div>
+
+<script id='adCode' type="text/javascript">
+  var pbjs = pbjs || {};
+  pbjs.que = pbjs.que || [];
+
+  function render(bid) {
+    const adContainer = document.getElementById('adContainer');
+    const adDisplayContainer = new google.ima.AdDisplayContainer(adContainer);
+
+    /* Prebid video ad unit */
+    const adsRequest = new google.ima.AdsRequest();
+    adsRequest.adTagUrl = '';
+    adsRequest.adsResponse = bid.vastXml;
+
+    adsLoader = new google.ima.AdsLoader(adDisplayContainer);
+    adsLoader.requestAds(adsRequest)
+  }
+
+  /* Prebid video ad unit */
+  const videoAdUnit = {
+    code: 'adContainer',
+    renderer: {
+      render,
+      url: "https://imasdk.googleapis.com/js/sdkloader/ima3.js"
+    },
+    mediaTypes: {
+      video: {
+        context: 'outstream',
+        playerSize: [640, 480],
+        mimes: ['video/mp4'],
+        protocols: [1, 2, 3, 4, 5, 6, 7, 8],
+        playbackmethod: [2],
+        skip: 1,
+      }
+    },
+    bids: [{
+      bidder: 'appnexus',
+      params: {
+        placementId: 13232385
+      }
+    }]
+  };
+
+  pbjs.que.push(() => {
+    pbjs.addAdUnits(videoAdUnit);
+
+    pbjs.requestBids({
+      bidsBackHandler: (bids) => {
+        const highestCpmBids = pbjs.getHighestCpmBids('adContainer');
+        pbjs.renderAd(document, highestCpmBids[0].adId);
+      }
+    });
+  });
+</script>
+<script>
+  /* Display ad code as example on page. */
```

Comment:
@bretg How about this instead of JS fiddle. Example code and interpreted code are both one and the same.